app/components/clock/Clock.jsx

```diff
@@ -33,7 +54,15 @@ class Clock extends React.Component {
     render() {
         return (
             <span className="clock">
-              {this.state.dateNow}
+            <span className="hours">{this.state.hours}:</span>
+            <span className="minutes">{this.state.minutes}:</span>
+            <span className="seconds">{this.state.seconds} </span>
+            {
+                this.props.hourFormat === '12' ? 
+                    <span className="formatHour">
+                        {this.state.hour >= 12 ? 'PM' : 'AM'}
```

Comment:
As I said somewhere above about *introduced separate variable if you see that function calls twice or more*, - same true for *variables*. `this.state.hours` is used twice here (in `render`) (by the way, you've made typo, there is no `this.state.hour` property).
Moreover, it is good practice to use destructor to extract properties at the beginning of the method, which uses `this.state` or `this.props`. Like this:

```javascript
const { hours, minutes, seconds } = this.state;
...
<div>{ hours }</div>
...
```